packages/webview_flutter/webview_flutter_android/example/pubspec.yaml

```diff
@@ -33,3 +33,8 @@ flutter:
     - assets/sample_video.mp4
     - assets/www/index.html
     - assets/www/styles/style.css
+
+# FOR TESTING AND INITIAL REVIEW ONLY. DO NOT MERGE.
+# See https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changing-federated-plugins
+dependency_overrides:
+   {webview_flutter_android: {path: ../../../webview_flutter/webview_flutter_android}, webview_flutter_platform_interface: {path: ../../../webview_flutter/webview_flutter_platform_interface}}
```

Comment:
This can be removed and the minimum version of the  `webview_flutter_platform_interface` should be bumped to `^2.7.0`.

```suggestion
```